README.md

```diff
@@ -18,6 +18,92 @@ To get updates, watch the repo and follow the [Twitter account](https://twitter
 
 # Tips list
 
+## #01(number) - Separate name and surname from fullname string.
+
+> yyyy-mm-dd(date) by [@0xmtn](https://github.com/0xmtn/)
+
+Sometimes, for the sake of UI, it is a neccessity to have a full control over a fullname when displaying it.
+
+```javascript
+Array.prototype.abbrFullName = function(name, surname){ 
+    return this.map(function(elem, idx, arr){
+        return idx != 0 && idx != arr.length-1 ? elem[0]+"." : 
+            (name && idx==0) || (surname && idx==arr.length-1) ? elem[0]+"." : 
+            elem;
+    }).join(" ");
+}
+```
+Examples:
+
+```javascript
+var fullname_str = "John Ronald Reuel Tolkien";
+
+var fullname_word_list = fullname_str.split(" ");
+console.log(fullname_word_list);
+// ["John", "Ronald", "Reuel", "Tolkien"]
+
+fullname_word_list.abbrFullName(false, false); //abbreviate only the middle names
+// "John R. R. Tolkien"
+
+fullname_word_list.abbrFullName(true, false); //abbreviate name too, but not surname
+// "J. R. R. Tolkien"
+
+fullname_word_list.abbrFullName(false, true); // abbreviate surname too, but not name
+// "John R. R. T."
+
+fullname_word_list.abbrFullName(true, true); // abbreviate name and surname too
+// "J. R. R. T."
+```
+
+## #01(number) - Shuffle an Array
+
+> yyyy-mm-dd(date) by [@0xmtn](https://github.com/0xmtn/)
+
+This snippet here uses [Fisher-Yates Shuffling](https://www.wikiwand.com/en/Fisher%E2%80%93Yates_shuffle) Algorithm to shuffle a given array.
+  
+```javascript
+function shuffle(arr){
+    for (var i = arr.length - 1; i > 0; i--) {
+        var j = Math.floor(Math.random() * (i + 1));
+        var temp = arr[i];
+        arr[i] = arr[j];
+        arr[j] = temp;
+    }
+    return arr;    
+}
+```
+An example:
+
+```javascript
+a=[1,2,3,4,5,6,7,8];
+b = shuffle(a);
+console.log(b);
+//[2, 7, 8, 6, 5, 3, 1, 4]
+```
+
+Also, it can be used as a function in Array.prototype like this:
+
+```javascript
+Array.prototype.shuffle = function(){
+    for (var i = this.length - 1; i > 0; i--) {
```

Comment:
Despite the fact that they are both identical, sure it is more convenient for the eyes to move `var` out of the `for`.